features/admin.sms-providers.v1/pages/vonage-sms-provider.tsx

```diff
@@ -43,6 +44,21 @@ const VonageSMSProvider: FunctionComponent<VonageSMSProviderPageInterface> = (
     } = props;
 
     const { t } = useTranslation();
+    const [ isShow, setIsShow ] = useState(false);
```

Comment:
Shouldn't this functionality come from the underlying password input component?

Reply:
We are using `FinalFormField` here, and since the password input component does not provide this capability, we are adding it as an `InputAdornment`.

Reply:
> since the password input component does not provide this capability

That the issue. We need to fix this in the password input component 🙃